src/Microsoft.AspNetCore.Mvc.Core/ApplicationModels/ActionModel.cs

```diff
@@ -123,5 +124,20 @@ public ActionModel(ActionModel other)
         string ICommonModel.Name => ActionName;
 
         public IList<SelectorModel> Selectors { get; }
+
+        public string DisplayName
```

Comment:
Can you also add something similar to the other model types, and add a `DebuggerDisplay`? This is a small enhancement but would prevent the need for a follow up.